### PR TITLE
Add build logic for Bottlerocket AWS variants

### PR DESCRIPTION
**Pull request by abhay-krishna**
This PR adds the Bottlerocket AWS variants to build/release CI.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: build/lib/validate_artifacts.sh [hunk 1]
```diff
@@ -35,6 +35,11 @@ if [ -n "$IMAGE_FORMAT" ]; then
   fi
 fi
 
+if ! stat $EXPECTED_FILES_PATH &> /dev/null; then
```
Comment (jaxesn) [inline build/lib/validate_artifacts.sh:38]: this is risky, if we forget to add the file it will go unnoticed....

Comment (abhay-krishna) [reply build/lib/validate_artifacts.sh:38]: Right. I'll try to skip it just for this case

Comment (abhay-krishna) [reply build/lib/validate_artifacts.sh]: Yeah I fixed it, it was a silly thing

**Review by abhinavmpandey08 — APPROVED** [review]
/approve
/lgtm
/woof

**Pull request merged**